docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -178,38 +188,39 @@ return (
 
 {{"demo": "RowGroupingUseKeepGroupedColumnsHidden.js", "bg": "inline", "defaultCodeOpen": false}}
 
-## Disable the row grouping
+## Disable row grouping
 
 ### For all columns
 
-You can disable row grouping by setting the `disableRowGrouping` prop to true.
-
-It will disable all the features related to the row grouping, even if a model is provided.
+To disable row grouping for all columns, set the `disableRowGrouping` prop to `true`.
+This disables all features related to row grouping, even if a model is provided.
```

Comment:
This statement seems to be wrong, as mentioned in the section below (grouping non..... programatically)
```suggestion
This disables all features related to row grouping, unless a model is provided, in which case the row grouping works in a read-only mode as detailed in [this](#grouping-non-groupable-columns) section.
```

Reply:
```suggestion
This disables all features related to row grouping, unless a model is provided, in which case row grouping is set to read-only mode as described in [Grouping non-groupable columns programmatically](#grouping-non-groupable-columns-programmatically).
```